metadata-models/src/main/pegasus/com/linkedin/ermodelrelation/ERModelRelationProperties.pdl

```diff
@@ -0,0 +1,79 @@
+namespace com.linkedin.ermodelrelation
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.CustomProperties
+
+/**
+ * Properties associated with a ERModelRelation
+ */
+@Aspect = {
+  "name": "eRModelRelationProperties"
```

Comment:
let's update the name (`Relation` -> `Relationship`) and casing here to look like this: `erModelRelationshipProperties`

and then update the references to that as well